apps/coscrad-frontend/src/components/higher-order-components/selected-resources.container.tsx

```diff
@@ -39,12 +38,15 @@ export const SelectedResourceContainer = ({
             />
         );
     }
+    if (loadableResources.errorInfo) return <ErrorDisplay {...loadableResources.errorInfo} />;
```

Comment:
We should find a way to wrap in this behaviour. We may need to update our `displayLoadable...` util.